File: front/kaemeweb/src/app/modules/shared/models/PedidoModels/pedido-search.ts
export interface PedidoSearch {
  num_Pedido?: string | null;
  cliente_Id?: string | null;
  usuario_Id?: string | null;
  data_Inicio_Pedido?: Date | null;
  data_Fim_Pedido?: Date | null;
  data_Inicio_Pagamento?: Date | null;
  data_Fim_Pagamento?: Date | null;
  status_Pedido_Id?: Date | null;
  pago?: boolean | null;
  tipo_Pagamento_Id?: string | null;
}

File: back/KaemeWebApi/Dominio/Models/PedidoModels/PedidoSearch.cs
namespace Dominio.Models
{
    public partial class PedidoSearch
    {
        public string? Num_Pedido { get; set; }
        public string? Cliente_Id { get; set; }
        public string? Usuario_Id { get; set; }
        public DateTime? Data_Inicio_Pedido { get; set; }
        public DateTime? Data_Fim_Pedido { get; set; }
        public DateTime? Data_Inicio_Pagamento { get; set; }
        public DateTime? Data_Fim_Pagamento { get; set; }
        public string? Status_Pedido_Id { get; set; }
        public bool? Pago { get; set; }
        public string? Tipo_Pagamento_Id { get; set; }

    }
}